connector/saml/saml.go

```diff
@@ -466,21 +466,25 @@ func (p *provider) validateConditions(assertion *assertion) error {
 		}
 	}
 	// Validates audience
+	audienceValue := p.entityIssuer
```

Comment:
This works for me, but do you think it'd be better to make entityIssuer required? I'm a little on the fence, but it seems like this is important when verifying the audience restrictions.

Would like to know what sounds more reasonable to you.